opendevin/config.py

```diff
@@ -5,12 +5,29 @@
 
 load_dotenv()
 
+DEFAULT_CONFIG = {
+    "LLM_API_KEY": None,
+    "LLM_BASE_URL": None,
+    "WORKSPACE_DIR": os.path.join(os.getcwd(), "workspace"),
+    "LLM_MODEL": "gpt-4-0125-preview",
+    "SANDBOX_CONTAINER_IMAGE": "ghcr.io/opendevin/sandbox",
+    "RUN_AS_DEVIN": "false",
+    "LLM_EMBEDDING_MODEL": "local",
+    "LLM_NUM_RETRIES": 6,
+    "LLM_COOLDOWN_TIME" : 1,
+    "DIRECTORY_REWRITE" : "",
+    "PROMPT_DEBUG_DIR": "",
+}
+
 config_str = ""
 if os.path.exists("config.toml"):
     with open("config.toml", "rb") as f:
         config_str = f.read().decode("utf-8")
 
 config = toml.loads(config_str)
+for key, value in DEFAULT_CONFIG.items():
+    if key not in config:
+        config[key] = value
```

Comment:
Am I reading this right, config doesn't save, thus get() doesn't return, variables saved _only_ in .env, unless they're also in toml in which case toml overrides? That's a bit different than get_or_default() or get_or_none(), which read .env.

Would it be a problem to continue accepting .env as source of values?

Reply:
Convention would be for ENV vars to override config file values. We might want to mirror that.

Reply:
> Am I reading this right, config doesn't save

What do you mean not save?

> Convention would be for ENV vars to override config file values. We might want to mirror that.

According to previous logic, config file should override env.
```python
def _get(key: str, default):
    value = config.get(key, default)
    if not value:
        value = os.environ.get(key, default)
    return value
```

Reply:
I mean, the values from .env were not read and stored in config, therefore get() would ignore them.

Reply:
Thanks, I find this bug and already fix it in the last commit.